GNUmakefile

```diff
@@ -53,6 +53,7 @@ ALL_TARGETS = linux_386 \
 	linux_amd64 \
 	linux_arm \
 	linux_arm64 \
+	linux_s390x \
```

Comment:
wouldn't adding this line cause s390x builds to happen in the oss pipeline?

Reply:
no, this only lists `linux_s390x` as one of the potential `make pkg/` targets. You can build an s390x binary manually this way, but the oss build pipeline depends on the build workflow. (attached a screenshot of a running build pipeline for this branch)
![Screenshot 2023-07-26 at 16 52 03](https://github.com/hashicorp/nomad/assets/470696/8610a4a5-fa0d-4ece-a7ae-3137e261d8c2)